integ/components/deadline/deadline_01_repository/bin/deadline_01_repository.ts

```diff
@@ -17,17 +19,23 @@ const integStackTag = process.env.INTEG_STACK_TAG!.toString();
 
 const componentTier = new Stack(app, 'RFDKInteg-DL-ComponentTier' + integStackTag, {env});
 
+// This will get the installers for the latest version of Deadline
+const version = new VersionQuery(componentTier, 'VersionQuery');
```

Comment:
The integ tests currently allow for testing arbitrary versions of Deadline (not just latest) through definition of the `DEADLINE_VERSION` environment variable. I'm thinking that we should probably use/reference that in these additions as well. Reason being tha it's possible that the container images will be built from a different version of Deadline than latest (they're based on the staged version), and we'd want the version that we're passing to the constructs to match that version.

I do like the idea of getting the Repository's version from the VersionQuery in our tests -- makes sure that it works. 

What do you think?

Reply:
I was looking at my changes to the integ tests and I was thinking that rather than doing the VersionQuery at this point and injecting it into the other structs, it would make more sense to pull the ThinkboxDockerRecipes up to this level and inject that instead. That way the version used in the VersionQuery would match the version used by the recipes, which is defined with `DEADLINE_VERSION`. This is how the example apps were changed to work.